packages/coin-support-utils/tests/02.createAccount/01.createDerivationPathGenerator/index.ts

```diff
@@ -0,0 +1,14 @@
+import { describe, test } from '@jest/globals';
+import { createDerivationPathGenerator } from '../../../src/createAccount';
+import { fixtures } from './__fixtures__';
+
+describe('01. Create Derivation Path Generator', () => {
+  fixtures.valid.forEach(({ name, input, output }) => {
+    test(name, () => {
+      const generator = createDerivationPathGenerator(input.basePath);
+      const result = generator(input.existingDerivationPaths, input.limit);
+      expect(generator).toBeInstanceOf(Function);
```

Comment:
Check if generator is a function before invoking it

```suggestion
      expect(generator).toBeInstanceOf(Function);
      const result = generator(input.existingDerivationPaths, input.limit);
```